Respawn/Checkpoint.cs

```diff
@@ -152,17 +152,22 @@ private class Relationship
                     {
                         if (!await reader.IsDBNullAsync(0))
                         {
-                            tables.Add("\"" + reader.GetString(0) + "\".\"" + reader.GetString(1) + "\"");
+                            tables.Add(CorrectSqlQuotes("\"" + reader.GetString(0) + "\".\"" + reader.GetString(1) + "\""));
                         }
                         else
                         {
-                            tables.Add("\"" + reader.GetString(1) + "\"");
+                            tables.Add(CorrectSqlQuotes("\"" + reader.GetString(1) + "\""));
                         }
                     }
                 }
             }
 
             return tables.ToList();
         }
+
+        private string CorrectSqlQuotes(string input)

```

Comment:
Why do we  need to correct the quotes? Should the initial string just be built correctly with the right quoted identifiers?

Reply:
I did it for clarity, the strings look a bit hard to read when you add the quote characters which was why I went down this path but I see your point, but I've tweaked the code to remove the helper method and moved the quote character inline wherever it is used.